From 3361ee542c6ebb9dc479d2b194dc1141b17df5d3 Mon Sep 17 00:00:00 2001
From: Xiaolei Wang <xiaolei.wang@windriver.com>
Date: Sat, 9 May 2020 11:14:15 +0800
Subject: [PATCH] arm: dts: add kim and btwilink device tree for am335x
 bluetooth

add kim and btwilink device tree for am335x bluetooth

Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 arch/arm/boot/dts/am335x-evm.dts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-evm.dts b/arch/arm/boot/dts/am335x-evm.dts
index 67bc672e6360..66f422d45efe 100644
--- a/arch/arm/boot/dts/am335x-evm.dts
+++ b/arch/arm/boot/dts/am335x-evm.dts
@@ -154,6 +154,18 @@
 		};
 	};
 
+	kim {
+                compatible = "kim";
+                nshutdown_gpio = <117>; /* Bank3, pin21 */
+                dev_name = "/dev/ttyS1";
+                flow_cntrl = <1>;
+                baud_rate = <3000000>;
+        };
+
+        btwilink {
+                compatible = "btwilink";
+        };
+
 	sound {
 		compatible = "simple-audio-card";
 		simple-audio-card,name = "AM335x-EVM";
-- 
2.17.1

